packages/documentation/src/shared/snapshots/schemes.ts

```diff
@@ -17,8 +17,6 @@ export function schemes(renderFn: (scheme: string) => TemplateResult, options: I
   return html`${[...additionalSchemes, ...Object.values(COLOR_SCHEMES)]
     .filter(filter)
     .map(
-      scheme => html` <div data-color-scheme="${scheme}" class="p-16 palette-default">
-        ${renderFn(scheme)}
-      </div>`,
+      scheme => html` <div data-color-scheme="${scheme}" class="p-16">${renderFn(scheme)}</div>`,
```

Comment:
It would be good to test how the palettes behave without a defined `data-color-scheme`, I have seen this article about dark mode in Cypress: https://www.cypress.io/blog/test-your-web-app-in-dark-mode

Reply:
You're right, we should test preferred colors too and using chrome with the command line flag `--force-dark-mode=true` could be an easy solution to do so.